### DISCUSSION TITLE
Github Actions to Test

### DISCUSSION THREAD

**Discussion by jrock2004** [nicknisi/dotfiles#187] [Ideas]
So I just learned that you can have macOS so you can test this all works.

![image](https://user-images.githubusercontent.com/655716/128417764-e49cc876-36c7-4c11-9b3d-f240ff175ba4.png)

Now I am thinking about how to add some tests to my install scripts and now on PR before merging I can be sure the changes will work on all my supported OS. Is this overkill?

**Comment by nicknisi** [comment]
This is an awesome idea! Did you make any progress towards it? (I only now discovered this discussion because I must've had my GH notifications set wrong).

**Reply by jrock2004** [reply]
I need to get back to this . I think I was having some issues
